Syndication/AzureStack.MarketplaceSyndication.psm1

```diff
@@ -692,18 +689,15 @@ function Import-ByDependency
 function Test-AzSOfflineMarketplaceItem {
     param (
         [parameter(mandatory = $true)]
-        [String] $Destination,
-
-        [parameter(mandatory = $true)]
-        [String] $ArmEndpoint,
-
-        [parameter(mandatory = $true)]
-        [String] $SubscriptionId
+        [String] $Destination
     )
 
     $ctx = Get-AzureRmContext
     $AccessToken = Resolve-AccessToken -Context $ctx -AccessToken $AccessToken
     $headers = @{ 'authorization' = "Bearer $AccessToken"}
+    $ArmEndpoint = $ctx.Environment.ResourceManagerUrl
```

Comment:
Same here. You can uither use camel casing or pascal casing, but please do not mix them in one script.